app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) != 3 or parts[0] != "cp":
+        return
+
+    _, source_file, destination_file = parts
+
+    if source_file == destination_file:
```

Comment:
Still not optimal - you have two  unnecessary return instructions. Try to follow this:
check if len of command is correct, unpack command, check if command is correct, proceed with the rest of logic.
Think how you can write your conditional statements so that it if they are true you continue with execution and if they are not your code does nothing. this way you will not need to add extra return instructions